Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM node:10.7.0-stretch as builder
+WORKDIR /src
+
+COPY client client
+COPY server server
+COPY package.json package.json
+COPY package-lock.json package-lock.json
+COPY webpack.config.prod.js webpack.config.prod.js
+COPY elm-package.json elm-package.json
+RUN npm install && npm run build
+
+FROM registry.opensource.zalan.do/stups/node:8.9.4-alpine-34
+MAINTAINER Sergii Kamenskyi <sergukam@sergukam.com>
+LABEL Description="This Nakadi UI does not check auth by default. Please use Auth plugins for use in production."
+
+WORKDIR /app
+COPY --from=builder /src/dist dist
```

Comment:
It would be nice to copy only one time, if possible, for example `COPY --from=builder /src/ .`

Reply:
the src dir contains a lot of trash (build system modules, tmp build artifacts, etc ) that we don't need.
I want to have the smallest and cleanest container possible.